### PR TITLE
Adding option for editing relation collection - bootstrap_collection

### PR DESCRIPTION
**Pull request by vladimirdjuricic**

### CODE REVIEW

**Comment by rvanlaarhoven** [comment]
Restarted the failed build due to the Github DDoS issues.

File: Form/Type/CrudRelationType.php [hunk 1]
```diff
@@ -102,15 +101,38 @@ public function addRelations(FormBuilderInterface $builder)
...
     /**
-     * Check if the property is allowed
+     * Add "Edit 'relation' button" with relation group.
+     *
+     * @param FormBuilderInterface $builder
+     * @param array                $relation
+     */
+    public function addEditRelationCollection(FormBuilderInterface $builder, $relation)
+    {
+        $builder->add('Edit '.$relation['fieldName'], 'button', array(
```
Comment (rvanlaarhoven) [inline Form/Type/CrudRelationType.php]: Please use square brackets for arrays

Comment (vladimirdjuricic) [inline Form/Type/CrudRelationType.php]: sorry, missed that one...
fixed

Comment (rvanlaarhoven) [inline Form/Type/CrudRelationType.php]: :no_good:

**Pull request merged**